internal-registry/redhat-developer/web-terminal/nightly/meta.yaml

```diff
@@ -23,9 +23,7 @@ spec:
       cookiesAuthEnabled: true
   containers:
   - name: web-terminal
-    image: "quay.io/eclipse/che-machine-exec:nightly"
-    #    ^ it'll be replaced with new image below after https://github.com/eclipse/che/issues/16942 is fixed
-    #    image: "quay.io/redhat-developer/web-terminal:nightly"
+    image: "${RELATED_IMAGE_plugin_redhat_developer_web_terminal_nightly}"
```

Comment:
Is there a strong reason to set RELATED IMAGE for nightly version?
If we use related images for airgap mode - then we replace that value with digest, and nightly is not nightly anymore.

Reply:
In general, once we start using digests stuff like "nightly" and "latest" are near-impossible to support. I don't know the preferred practice for this -- @nickboldt any ideas here?

I don't see a good way of avoiding ErrImagePull if we leave these images as tags. Also, once this image is set to a reg.redhat.io build, nightly will cease to exist anyways (unless we do releases every day). I'm not sure the best way to support this on the community front (except that leaving the env-var as is doesn't change functionality).